Documentation/operation docs rework (#3074)
scality/metalk8s: docs/installation/troubleshooting/index.rst

.. _Troubleshooting Installation Guide:

Troubleshooting
===============

> This guide covers some of the common issues users face during and

Reviewer: ```suggestion
This section describes common issues users face during and
```

> after a MetalK8s installation. If your issue does not figure in this guide,

Reviewer: ```suggestion
after a MetalK8s installation. If your issue is not presented here,
```

> reach out to Scality support, or create a

Reviewer 2: We should have separate paragraphs, one for customers and one for the open source version:
If your issue is not presented in this guide, contact Scality Support.
If your issue is not presented in this guide, create a Github issue.

Reviewer 3: Agreed. You can use:

```rst
.. only:: scality-product

   Some paragraph to show on documentation.scality.com.
   
.. only:: not scality-product

   Some paragraph to show on other sources (RTD, documentation-internal, from the UI...).
```
